### PR TITLE
Clarify configuring and migrating  HelmChart CR

### PR DESCRIPTION
**Pull request by paigecalvert**
Update the headings of the existing Configuring topic and remove the information that is specific to migrating existing installations: https://deploy-preview-1616--replicated-docs.netlify.app/vendor/helm-native-v2-using

Moved the info about migrating existing installations to a new topic: https://deploy-preview-1616--replicated-docs.netlify.app/vendor/helm-v2-migrate

### CODE REVIEW

File: docs/vendor/helm-v2-migrate.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Migrating Existing Installations to HelmChart v2
```
Comment (paigecalvert) [inline docs/vendor/helm-v2-migrate.md:1]: This is a short topic, but there is so much content in the existing Configuring topic that it felt like information overload and hard to figure out what's what.

Here, it's more clearly stated that it's about migrating existing installations _after_ you configure v2

File: docs/vendor/helm-v2-migrate.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+This topic provides guidance for migrating existing installations that were deployed with the HelmChart custom resource `kots.io/v1beta1` with `useHelmInstall: true` to use `kots.io/v1beta2` instead.
+
+## Overview
+
+The HelmChart custom resource `kots.io/v1beta1` is deprecated and is not recommended for new installations. Existing installations that were installed with `kots.io/v1beta1` and `useHelmInstall: true` can be migrated to `kots.io/v1beta2` without needing to reinstall the application. For information about how to support installation with the `kots.io/v1beta2` HelmChart custom resource, see [Configuring the HelmChart Custom Resource v2](helm-native-v2-using).
```
Comment (cbodonnell) [inline docs/vendor/helm-v2-migrate.md:7]: Not sure if it would make this section too wordy, but I could see a small mention of _what_ you need to configure for the `kots.io/v1beta2` HelmChart custom resource being useful. Something like "For information about how to support installation with the `kots.io/v1beta2` HelmChart custom resource, including image re-writes, injecting pull secrets, and adding backup labels, see ..."

File: docs/vendor/helm-v2-migrate.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+### `useHelmInstall: false` Limitation
+
+Migration from `kots.io/v1beta1` with `useHelmInstall: false` to any other installation method (including to `kots.io/v1beta1` with `useHelmInstall: true` or to `kots.io/v1beta2`) is _not_ supported.
+
+To change the installation method from `useHelmInstall: false` to a different method, the user must reinstall your application in a new environment.
```
Comment (ajp-io) [inline docs/vendor/helm-v2-migrate.md:13]: I think it could probably be the same environment, and this is less words anyway.
```suggestion
To change the installation method from `useHelmInstall: false` to a different method, the user must reinstall your application.
```

**Review by ajp-io — DISMISSED** [review]

File: docs/vendor/helm-v2-migrate.md (hunk 2 above)
Comment (paigecalvert) [reply docs/vendor/helm-v2-migrate.md]: Nice, I like it
